### ISSUE TITLE
Date functions in SQLite

### ISSUE THREAD

**Issue by Nemo64** [beberlei/DoctrineExtensions#210]
At the very basic, I need to use the `Date` Function in dql. I use Mysql in production and sqlite for testing.
But I ran into the issue that the function name is not converted. It doesn't matter if I use `DATE`, `YEAR`, `MONTH` etc, they all have the same issue. I get these errors:

`SQLSTATE[HY000]: General error: 1 no such function: DATE`
`SQLSTATE[HY000]: General error: 1 no such function: YEAR`
`SQLSTATE[HY000]: General error: 1 no such function: MONTH`

So basically, the function is not converted into the `strftime` as expected.

I'm using doctrine orm and dbal `v2.5.4` in a symfony 2.8 instance with php 5.6.

Here an Example query: `app/console doctrine:query:dql "SELECT DATE_FORMAT(d.value, '%Y') FROM HnGenericTableBundle:TableValueDate AS d"`

**Comment by stevelacey** [comment]
Are you sure you've registered the types correctly?

There are tests for DATE_FORMAT that are passing.

https://github.com/beberlei/DoctrineExtensions/blob/master/tests/Query/Sqlite/DateFormatTest.php
